### DISCUSSION TITLE
Deletion of .received files after accepting changes in WinMerge

### DISCUSSION THREAD

**Discussion by mickvikt** [VerifyTests/Verify#629] [Q&A] [answered]
Hello,
I have a question: should .received files be deleted automatically after accepting changes in the diff tool (WinMerge) and saving the .verified file? In my case .received file remains alongside the .verify file.

Is there any way to automate this?

**Comment by SimonCropp** [comment, marked as answer]
have u tried diffenginetray https://github.com/VerifyTests/DiffEngine/blob/main/docs/tray.md  ?
